connectors/package-lock.json

```diff
@@ -90,6 +90,7 @@
         "eslint-plugin-simple-import-sort": "^12.1.0",
         "eventsource-parser": "^1.1.1",
         "hot-shots": "^10.0.0",
+        "htmlparser2": "^9.1.0",
```

Comment:
Can be removed.

Reply:
@flvndvd `types` does depend on `htmlparser2` right? Or did we remove the dependency?

Reply:
Yes `types` does, but `connectors` should not need it.

Reply:
As per IRL: this is in `../types` deps